README.md

```diff
@@ -28,6 +28,8 @@ Supported password/key types:
   * `ec521` - generates ECC P-521 private key
   * `rsa2048` - generates 2048-bit RSA private key
   * `rsa4096` - generates 4096-bit RSA private key
+  * `x25519` - generates x25519 (also known as curve25519) ECC private key
```

Comment:
I understand the guy's use-case from #10, but what is this mode for? x25519's majority use-case is ephemeral DH.

Reply:
Well, it may be used in the original `nacl`/`libsodium` `crypto_box` implementation for example